src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -380,10 +380,10 @@ private async Task<bool> CanAccessAllCiphersAsync(Guid organizationId)
             return true;
         }
 
-        // Provider users can only access all ciphers if RestrictProviderAccess is disabled
+        // Provider users can access all ciphers in V1 (to change later)
```

Comment:
⛏️ We can remove the "in V1 (to change later)" in the comment since its the intended behavior long term.

Reply:
Updated here 5000b67c6886d72545ec37efbd0a975ffc9dcc30